source/simulator/assemblers/newton_stokes.cc

```diff
@@ -346,7 +343,7 @@ namespace aspect
                 data.local_rhs(i) += (scalar_product(elastic_outputs->elastic_force[q],Tensor<2,dim>(scratch.grads_phi_u[i])))
                                      * JxW;
 
-              if (enable_additional_stokes_rhs)
+              if (enable_prescribed_dilation)
```

Comment:
This and the same change in line 356 is the only functional difference, right?

Reply:
Yes